nixos/doc/manual/development/running-nixos-tests-interactively.section.md

```diff
@@ -24,6 +24,23 @@ back into the test driver command line upon its completion. This allows
 you to inspect the state of the VMs after the test (e.g. to debug the
 test script).
 
+## Separating outputs {#sec-nixos-test-separating-outputs}
+
+Logs from the running virtual machines get output in-line and can be disruptive
+while using the python REPL. You can separate these streams by redirecting
+stderr:
```

Comment:
The documentation team recommends one sentence per line, which can be a long line. It works better with GitHub suggestions, as it requires no reflowing of these ignored line breaks.
(Pre-existing problem and not urgent for these additions)